packages/pg-typed/src/index.ts

```diff
@@ -408,18 +426,19 @@ function getTable<TRecord, TInsertParameters>(
   serializeValue: (columnName: string, value: unknown) => unknown,
 ) {
   return (
-    queryable: Queryable | undefined = defaultConnection,
+    connections: Queryable | Queryable[] | undefined = defaultConnection,
   ): Table<TRecord, TInsertParameters> => {
-    if (!queryable) {
+    if (!connections || (Array.isArray(connections) && !connections.length)) {
       throw new Error(
-        'You must either provide a "defaultConnection" to pg-typed, or specify a connection when accessing the table.',
+        'You must either provide a "defaultConnection" to pg-typed, or specify a list of connections when accessing the table.',
```

Comment:
The list of connections approach is still an advanced use case, so the error should still only tell you about the missing connection.

```suggestion
        'You must either provide a "defaultConnection" to pg-typed, or specify a connection when accessing the table.',
```